privacyidea/lib/resolvers/LDAPIdResolver.py

```diff
@@ -555,6 +555,8 @@ def getUserList(self, searchDict):
             # Simple fix for ignored sizelimit with Active Directory
             if len(ret) >= self.sizelimit:
                 break
+            if entry['type'] == 'searchResRef':
```

Comment:
Do we always have a type, even in case of OpenLDAP and eDirectory?
...or should we do

    if entry.get("type") == 'searchResRef:

Because, if there was no type, the complete LDAP search would break due to the exception and we would bailed out to the upper level and would get no result at all.